docs/reference/content/guides/querying.md

```diff
@@ -0,0 +1,250 @@
++++
+title = "Querying"
+[menu.main]
+  parent = "Reference Guides"
+  weight = 86
+  pre = "<i class='fa fa-file-text-o'></i>"
++++
+
+# Querying
+
+Using the Java driver directly involves [creating](http://mongodb.github.io/mongo-java-driver/3.0/driver/reference/crud/#crud) a lot of 
+`Document` instances to capture a query.   Morphia, on the other hand, offers a fluent API with which to build up a query and attempts to 
+provide as much type safety and validation as possible.  To this end, Morphia offers the `Query<T>` class which can be parameterized to 
+the type of your entity.
+
+## Creating a Query
+
+The `Datastore` is the key class when using Morphia.  Virtually all operations begin with the `Datastore`.  To create the `Query`, we 
+invoke the following code:
+
+```java
+Query<Product> query = datastore.createQuery(Product.class);
+```
+
+`createQuery()` returns an instance of `Query` with which we can build a query.
+
+### `filter()`
+
+The first method of interest is `filter()`.  This method takes two values: a condition string and a value.  The `value` parameter is, of 
+course, the value to use when applying the `condition` clause.  The `condition` parameter is a bit more complicated.  At its simplest, 
+the condition is just a field name.  In this case, the condition is assumed to be an [equality]({{< docsref "reference/operator/query/eq/"
+ >}}) check.  There is a slightly more complicated variant, however.
+ 
+The `condition` value can also contain an operator.  For example, to compare a numeric field against a value, you might write something 
+like this:
+
+```java
+query.filter("price >=", 1000);
+```
+
+In this case, we're instructing Morphia to add a filter using [$gte]({{< docsref "reference/operator/query/gte/" >}}).  This would result
+ in a query that looks like this:
+ 
+```javascript
+{ price: { $gte: 1000 } }
+```
+
+The list of supported filter operations can be found in the [FilterOperator]({{< srcref
```

Comment:
I debated that but wasn't sure how best to represent that.  I'll play with some options.